elasticdl/python/tests/test_module.py

```diff
@@ -34,8 +34,8 @@ def optimizer(lr=0.1):
     return tf.optimizers.SGD(lr)
 
 
-def eval_metrics_fn(predictions, labels):
-    return {"mse": tf.reduce_mean(tf.square(predictions - labels))}
+def eval_metrics_fn():
+    return {"mse": lambda labels, outputs: tf.square(outputs - labels)}
```

Comment:
I haven't looked closely yet but double check whether this is `labels, outputs` or `outputs, labels` (same in other examples)

Reply:
I have checked that here should be `labels, outputs`. 

Because this PR calculates evaluation metrics in the same way as Keras, we can refer to the "Custom metrics" section in this [link](http://faroit.com/keras-docs/2.0.4/metrics/).